app/src/main/java/com/tari/android/wallet/service/WalletService.kt

```diff
@@ -735,6 +735,7 @@ internal class WalletService : Service(), FFIWalletListenerAdapter {
             val response = tariRESTService.requestMaxTestnetTari(publicKeyHexString, requestBody)
             response.enqueue(object : Callback<TestnetTariAllocateMaxResponse> {
                 override fun onFailure(call: Call<TestnetTariAllocateMaxResponse>, t: Throwable) {
+                    Logger.e("1")
```

Comment:
?

Reply:
@CjS77 it's a debug note forgotten. Thanks, will delete 👍